manager/integration/tests/common.py

```diff
@@ -4992,6 +4992,17 @@ def cleanup_all_support_bundles(client):
     Clean up all support bundles
     :param client: The Longhorn client to use in the request.
     """
+    ret = get_core_api_client().list_pod_for_all_namespaces(
```

Comment:
Can you get `current-longhorn-version` setting instead of figuring out the version from LH pods?

Reply:
modified!